docs/pages/2020_Intro_Week/sections/data.rst

```diff
@@ -58,7 +58,7 @@ User
 
    The ``-a`` and ``-A`` flags we used above ensure that groups for all type strings and users are shown respectively.
 
-We can then inspect a groups contents, by the label (if it is unique) or the PK:
+We can then inspect a group's contents by its label (if it is unique) or the PK:
```

Comment:
`a group's content` or `the group's contents`? Also, not sure if we really need the `'s`.
My choice would be simply `the group content` or even better `the content of a group`

Reply:
Right. So `the content of a group` is equivalent to `a group's content`. But it's also "better English" to not use many apostrophes... I will change it to the latter and also reevaluate whether this section is talking about showing the content of a specific group or of groups in general.